test/CMakeLists.txt

```diff
@@ -1456,3 +1447,17 @@ add_custom_test(test_regression_opencl_float_mi100 GFX908_ENABLED VEGA_DISABLED
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  512, 28, 28 --weights 128, 512, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  64,  56, 56 --weights 256, 64,  1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
 )
+
+if(MIOPEN_TEST_GFX1030)
+    if(WORKAROUND_ISSUE_1053 AND MIOPEN_TEST_ALL)
+        list(APPEND SKIP_TESTS test_lrn_test)
+    endif()
+    if(WORKAROUND_ISSUE_1148 AND MIOPEN_TEST_ALL)
+        set_tests_properties(
+            test_conv_3d
+            test_conv_group
+            test_conv_extra
+            test_conv_for_implicit_gemm
```

Comment:
Please set properties of "custom" tests in `add_test_command`. AFAICS all these tests are "custom" ones.

PS Properties of executable tests should be set in in `add_test_executable`.

Reply:
@atamazov  could you shed more lights on how should I do as you suggested?
Yes, these tests are "custom" as added by "add_custom_test", also they share a label of "long tests".  I hope it can be guarded by "WORKAROUND_ISSUE_1148" so that it can be identified (more importantly, removed thereafter) more easily. Thanks!

Reply:
I think this should be something like 
```
    if(WORKAROUND_ISSUE_1148
       AND MIOPEN_TEST_GFX1030
       AND (${NAME} MATCHES "test_conv_3d"
            OR ${NAME} MATCHES "test_conv_group"
            OR ${NAME} MATCHES "test_conv_extra"
            OR ${NAME} MATCHES "test_conv_for_implicit_gemm" ))
        set_tests_properties(${NAME} PROPERTIES RUN_SERIAL On)
    endif()
```
inserted in `add_custom_test` between 
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/1a1df9fa05297d501efa9957778baed215baa873/test/CMakeLists.txt#L240-L242

Reply:
Oh, no, this should be inserted into `add_custom_test`!